templates/admin/awardnominations/view.html.twig

```diff
@@ -0,0 +1,52 @@
+{% extends '@Forumify/admin/base.html.twig' %}
+{% block title_page %}{% endblock %}
+{% block body %}
+    {% if data == null %}
+        <p>Unknown nomination!</p>
+    {% else %}
+        <h1 class="flex items-center gap-2 mb-4">        
+            {{ 'perscom.admin.award_nominations.view.title'|trans }} #{{data.nomination.id}}
+            {% include '@ForumifyPerscomPlugin/frontend/roster/components/status.html.twig' with {
+                status: data.statusItem,
+                class: 'text-small'
+            } only %}
+        </h1>
+        <div class="flex gap-4">
+            <div class="w-50">
```

Comment:
use grid instead of flex and w-x.

Most templates in this plugin were made before we had the grid/col css classes.